internal/configsource/envvarconfigsource/README.md

```diff
@@ -51,24 +51,18 @@ components:
 
 The typical case to use the environment variable config source is when one wants
 to inject YAML fragments. The example below shows how this can be done on Linux and
-Windows.
+Windows when running the collector from your current session. For guidance on setting
+service environment variables for your installation please see the related
+[Linux](../../../docs/getting-started/linux-installer.md#getting-started)
+and [Windows](../../../docs/getting-started/windows-installer.md#collector-configuration) installer documentation.
 
```

Comment:
https://github.com/signalfx/splunk-otel-collector/blob/main/docs/getting-started/linux-installer.md#getting-started does not have the steps to set service environment variables, I found them here instead https://github.com/signalfx/splunk-otel-collector/blob/main/docs/getting-started/linux-manual.md#collector-debianrpm-post-install-configuration

Reply:
thanks, updated